### DISCUSSION TITLE
Welcome to koar-item-editor Discussions!

### DISCUSSION THREAD

**Discussion by mburbea** [mburbea/koar-item-editor#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Torllay** [comment]
How can i increase the level of an item? I love the Shepard Set but i can't seem to increase it's defense values.

**Reply by mburbea** [reply]
Sadly level does not improve the defense of armor items. Try adding buffs that increase the armor amount.

**Reply by Torllay** [reply]
For example what buffs could i use? I'm relatively new to the game so i don't know the buffs name.

**Reply by mburbea** [reply]
Yeah there's probably around 7000 or so buffs, so your guess is as good as mine. I compiled them into a spreadsheet that might make it easier to find things:
https://docs.google.com/spreadsheets/d/1BY4MPB_K_Vci8EbGg8wgHSZqLccSdJfTd0akoZLakVE/edit#gid=1467112185

**Comment by MllnDllrMan** [comment]
Hey good day! Does this support PS4 saves? If so how do I start editing the save?

**Reply by mburbea** [reply]
You'll need a way to decrypt your save like save wizard. I know nothing of the specifics of it. I don't own the PS4 version but it's been asked numerous times.

**Reply by Torllay** [reply]
I think there are servers on Discord where you can buy a slot to decrypt your saves with Save Wizard for cheap instead of buying the software cause as far as i know the decryption happens server side of the software so you need to buy it.
Usually these people sell the slot for a month i think so you'd have a month of back and forth but i suggest you, in case you do it, to wait for your build or level to be maxed so you just set the perks and stuff you want in your final build once or a few times more and then pass it back to encrypted and have it work. It would be a bit messier if you wanna touch-up your save often like me for example since you'd have to hope the person you buy the slot for is free and can do it for you in that moment.

**Comment by BruunHero** [comment]
Would it be possible to add special items like Compass of Fate to the editor? Would be a cool way to add passive buffs to a character for repeat playthroughs.

**Comment by Drapsheik** [comment]
Hello, I am playing Re-Reckoning. While I scroll through the list of buffs I do not see all of them. When I try to add a buff through an ID from the list it does not work. It adds a buff with "none" as description. Did I do something wrong with the installation? Is there an installation?

Thank you,
Drap

**Comment by BakaKyuubi84** [comment]
@mburbea Would it be possible to add consumables to the editor?

I play on PS4 and use your editor for equipment. I have to say I absolutely love it, so thank you very much for all that you do on it.

**Reply by mburbea** [reply]
It's nothing I have planned. The only thing you can really modify without pain is quantity, and inventory flags (like stolen, transferable or stashable). The big issue is adding new items through the save file is tricky as the way data is stored in the save file is absurd. The best solution is, unfortunately, pc only and just use the console mod I wrote.

That said, if you want to create a new window as a PR to control quantity I accept PRs.
